openapi-v1.yaml

```diff
@@ -2729,79 +2729,304 @@ definitions:
+      license_id:
+        type: string
+        x-omitempty: true
+        description: License identifier from SPDX License List or Custom.
+      license_text:
         type: string
-    example:
-      parent: genomics-course
-      description: contains arrays and notebooks for an 101 course on genomics with TileDB
+        x-omitempty: true
+        description: License text
 
   GroupUpdate:
     description: Updates for a group. New values for the attributes.
     type: object
     properties:
+      description:
+        description: A human readable description of the content of the group
+        type: string
       name:
-        description: The new name of the group
+        description: The name of the group. If must be unique within the group.
         type: string
-      description:
-        description: A new human readable description of the content of the group
+      logo:
+        description: logo (base64 encoded) for the group. Optional
+        type: string
+      access_credentials_name:
+        description: the name of the access credentials to use. if unset, the default credentials will be used
         type: string
-    example:
-      name: intro-to-genomics
-      description: contains arrays and notebooks for an 101 course on genomics with TileDB
+      tags:
+        description: optional tags for groups
+        x-omitempty: false
+        type: array
+        items:
+          type: string
+          description: tag must be lowercase characters or hypen [a-z-]
+      license_id:
+        type: string
+        x-omitempty: true
+        description: License identifier from SPDX License List or Custom
+      license_text:
+        type: string
+        x-omitempty: true
+        description: License text
 
-  GroupListing:
-    description: The contents of a group i.e attributes, subgroups and assets
-    allOf:
-      - $ref: '#/definitions/Group'
-      - type: object
-        properties:
-          groups:
-            description: Contains one page of subgroups of the group.
-            type: array
-            x-omitempty: true
-            items:
-              $ref: "#/definitions/Group"
-          assets:
-            description: Contains one page of assets of the group as ArrayInfos
-            type: array
-            x-omitempty: true
-            items:
-              $ref: "#/definitions/ArrayInfo"
-          pagination_metadata:
-            $ref: "#/definitions/PaginationMetadata"
+  GroupMemberType:
+    description: File types that can be included in groups
+    type: string
+    enum:
+      # Group
+      - group
+      # Array
+      - array
+      # Notebook
+      - notebook
+      # UDF
+      - user_defined_function
+      # ML Model
+      - ml_model
+      # Files
+      - file
+
+  GroupMember:
+    description: A groups member, array or another groups, to add or remove from an existing group.
+    type: object
+    properties:
```

Comment:
This is missing `uri`

Reply:
Why `uri` is needed? This type is used to add/remove members that are already registered and thus accessible by namespace/name